dataverse-webapp/src/main/webapp/WEB-INF/templates/dataset/inputRenderer/textInputField.xhtml

```diff
@@ -7,8 +7,7 @@
     xmlns:jsf="http://xmlns.jcp.org/jsf">
 
     
-    <p:outputLabel styleClass="control-label" for="inputText" rendered="#{!hiddenLabel}">
-        #{datasetField.datasetFieldType.localeTitle}
+    <p:outputLabel styleClass="control-label" for="inputText" rendered="#{!hiddenLabel}" escape="false" value="#{datasetField.datasetFieldType.localeTitle}">
```

Comment:
There are other places to change:
`htmlMarkupInputField.xhtml` and `controlledVocabInputField.xhtml`

Reply:
Also: `editMetadata.xhtml`

Reply:
Hmm i think merging with develop has overwritten my changes in this places where also @wtr-icm has changed descriptions..